### DISCUSSION TITLE
Make it visually clearer which version of the docs is viewed

### DISCUSSION THREAD

**Discussion by itaysk** [aquasecurity/trivy#4352] [Ideas]
### Description

As suggested in https://github.com/aquasecurity/trivy/issues/2961#issuecomment-1546478210, and evident in https://github.com/aquasecurity/trivy/discussions/4211#discussioncomment-5842964
When users land in documentation from a direct link most likely this link will lead to an older version of the docs. We have the version switcher on top but 1) when scrolling it disappears, and when anchor-linking as well 2) even if it's visible it doesn't stand out.
Other projects like OPA added a scary banner whenever browsing older version https://www.openpolicyagent.org/docs/v0.51.0/
IDK if we should do the same but we should definitely think about how to emphasize the current doc version and the version selector.

### Target

None

### Scanner

None

**Comment by simar7** [comment]
Yeah good point. I had implemented this in the Postee docs and it can be done using this https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/

**Reply by knqyf263** [reply]
Looks great! I'll create an issue, and close this discussion.
